Pod/Classes/PINRemoteImageManager.h

```diff
@@ -116,6 +116,13 @@ typedef void(^PINRemoteImageManagerAuthenticationChallenge)(NSURLSessionTask *ta
 @property (nonatomic, readonly) PINCache *cache;
 
 /**
+ Creates and returns a PINRemoteImageManager created with the specified configuration. If configuration is nil, [NSURLSessionConfiguration defaultConfiguration] is used. You specify a custom configuration if you need to configure timeout values, cookie policies, additional HTTP headers, etc.
```

Comment:
I think this should be "Create and return" to match the style of the other comments.